src/sidebar/store/modules/filters.js

```diff
@@ -46,7 +46,7 @@ import { createStoreModule } from '../create-store';
  */
 
 /**
- * @typedef {Object.<FilterKey, FilterOption>} Filters
+ * @typedef {{user?: FilterOption}} Filters
```

Comment:
Should this not be `Record<FilterKey, FilterOption>`?

Reply:
Because we assign empty objects to filter there are two solutions:
* make the key optionals
* or define filter as `Record<FilterKey, FilterOption>|{}`

I went for the last option.